docs/monitoring.md

```diff
@@ -0,0 +1,25 @@
+# Use OpenShift Cluster Monitoring to expose Hive metrics
+
+OpenShift ships with monitoring stack that can be used to export various
+Hive metrics.
+
+## Enabling monitoring for user-defined projects
+
+Follow the guidelines as defined in [OpenShift documentation][user-projects-monitoring]
```

Comment:
By enabling monitoring for user-defined projects in OpenShift, we can export metrics even for `hive` namespace. So this makes the previously discussed requirement of `openshift-` prefix no longer required.